endpoints-framework-tools/src/main/java/com/google/api/server/spi/tools/EndpointsTool.java

```diff
@@ -48,12 +48,14 @@
   private final Map<String, EndpointsToolAction> actions;
 
   public EndpointsTool() {
-    actions = new LinkedHashMap<String, EndpointsToolAction>();
+    actions = new LinkedHashMap<>();
     actions.put(GetDiscoveryDocAction.NAME, new GetDiscoveryDocAction());
     actions.put(GetClientLibAction.NAME, new GetClientLibAction());
     actions.put(GenApiConfigAction.NAME, new GenApiConfigAction());
     actions.put(GenClientLibAction.NAME, new GenClientLibAction());
-    actions.put(GetSwaggerDocAction.NAME, new GetSwaggerDocAction());
+    actions.put(GetOpenApiDocAction.NAME, new GetOpenApiDocAction());
+    actions.put(GetOpenApiDocAction.LEGACY_NAME,
+        new GetOpenApiDocAction(GetOpenApiDocAction.LEGACY_NAME, false));
```

Comment:
The naming is a bit confusing, it's doing the same thing that yours is--hiding it from the command list help. If you ran the tool with just "get-swagger-doc" it would still show the individual command help.